repotracker/repotracker.go

```diff
@@ -923,6 +923,18 @@ func createVersionItems(ctx context.Context, v *model.Version, metadata model.Ve
 		"version": v.Id,
 	}))
 
+	grip.ErrorWhen(len(buildsToCreate) == 0, message.Fields{
+		"message":    "version has no builds",
+		"version":    v.Id,
```

Comment:
It seems that buildsToCreate is empty when tasks is always empty, or if the parser project has no buildvariants. It might be helpful to return some of that information; maybe during the loop above, we can store a map of buildvariant -> number of tasks?

Reply:
Oh yeah that's true. I mapped buildvariant to an error message instead, since there is potentially more detail there.